Model/Cron/Customers.php

```diff
@@ -72,29 +76,32 @@ public function getList($last_update)
                     }
                 }
 
+                $websiteId = (int) $this->storeManager->getStore($s['store_id'])->getWebsiteId();
                 // Get fields to sync from configuration page.
-                $sync_fields = $this->helperData->getGeneralConfig('fields');
+                $sync_fields = $this->helperData->getSmailyConfig('fields', $websiteId);
                 $sync_fields = explode(',', $sync_fields);
-    
+
                 // Create list with subscriber data.
                 $subscriberData = [
                     'email' => $s['subscriber_email'],
                     'name' => $customer ? ucfirst($customer->getFirstname()).' '.ucfirst($customer->getLastname()) : '',
                     'subscription_type' => 'Subscriber',
                     'customer_group' => $customer ? $this->helperData->getCustomerGroupName($customer->getGroupId()) : 'Guest',
                     'customer_id' => $customer_id,
+                    'website_id' => $websiteId,
                     'prefix' => $customer ? $customer->getPrefix() : '',
                     'firstname' => $customer ? ucfirst($customer->getFirstname()) : '',
                     'lastname' => $customer ? ucfirst($customer->getLastname()) : '',
                     'gender' => $customer ? ($customer->getGender() == 2 ? 'Female' : 'Male') : '',
-                    'birthday' => $DOB,
+                    'birthday' => $DOB
                 ];
 
                 // Standard values always collected for subscriber.
                 $subscriber = [
                     'email' => $subscriberData['email'],
                     'name' => $subscriberData['name'],
-                    'store' => $this->storeManager->getStore($s['store_id'])->getName()
+                    'store' => $this->storeManager->getStore($s['store_id'])->getName(),
+                    'website_id' => $subscriberData['website_id']
```

Comment:
Send Website Name to Smaily instead of its ID.

Reply:
I was relying on `website_id` in `cronSubscribeAll` to filter out subscribers by ID. I am now converting name to id for the evaluation. Let me know what you think. Should I compare website names already then?